### PR TITLE
style and clarifications of 0.21 test doc

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

File: docs/v0.21/testing-0.21rc2.md [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+SQLite was chosen as a new database because it provides certain guarantees necessary for ensuring that the wallet remains backwards compatible moving forward. Furthermore, unlike `BerkelyDB 4.8`, SQlite allows us to have a one file wallet instead of a wallet directory.
 
 ### 1. Preparation
-If you grabbed the binary for this release candidate, you're good to go. If you went down the source route, it is required that you installed the `sqlite3` dependency and compiled the source code with wallet functionality. Skip this section if you intentionality do not want wallet functionality, or don't want to test it. This guide will be split between using the GUI and the CLI. Follow whichever you are comfortable using.
+If you grabbed the binary for this release candidate, you're good to go. If you went down the source route, it is required that you installed the `sqlite3` dependency and compiled the source code with wallet functionality. Skip this section if you intentionally do not want wallet functionality or don't want to test it. This guide will be split between using the GUI and the CLI. Follow whichever you are comfortable using.
 
 ### 2. Manual Testing
```
Comment (adamjonas) [inline docs/v0.21/testing-0.21rc2.md:44]: Maybe a note on to trigger QT if compiling from source. Not everyone realizes there is a GUI. 😉

File: docs/v0.21/testing-0.21rc2.md [hunk 2]
```diff
@@ -96,33 +97,32 @@ $ tor
...
+We are going to set up our node to [listen on Tor automatically](https://github.com/bitcoin/bitcoin/blob/master/doc/tor.md#3-automatically-listen-on-tor). This means that the node is going to look for other peers on the Tor network.
 
 ---
 
 ## Testing Signet
-The Bitcoin [testnet](https://en.bitcoin.it/wiki/Testnet) is a [proof-of-work](https://en.bitcoin.it/wiki/Proof_of_work) based testing framework where volunteers are relied on to [mine](https://en.bitcoin.it/wiki/Mining) [blocks](https://en.bitcoin.it/wiki/Block) with real CPU power and in turn receive worthless testnet coins. Since the economics of the the mainnet are not at play here; we get a network that is unpredictable and, frustratingly, unreliable.
+The Bitcoin [testnet](https://en.bitcoin.it/wiki/Testnet) is a [proof-of-work](https://en.bitcoin.it/wiki/Proof_of_work) based testing framework where volunteers are relied on to [mine](https://en.bitcoin.it/wiki/Mining) [blocks](https://en.bitcoin.it/wiki/Block) with real CPU power and in turn receive worthless testnet coins. Since the economics of mainnet are not at play here, we get a network that is unpredictable and, frustratingly, unreliable.
```
Comment (adamjonas) [inline docs/v0.21/testing-0.21rc2.md:110]: I think the links here are overkill. You don't need to teach testers what mining or blocks are if they are in the trenches testing a release candidate.

File: docs/v0.21/testing-0.21rc2.md [hunk 3]
```diff
@@ -96,33 +97,32 @@ $ tor
...
 
-
-When you're node connects to the Bitcoin network, it makes at least [two outbound block-relay only connections](https://github.com/bitcoin/bitcoin/pull/15759). This release introduces [Anchor Connections](https://github.com/bitcoin/bitcoin/pull/17428). Anchors are the [two outbound block-relay connections]() your node is connected to; logged to an `anchors.dat` file so that they can be used upon a node restart. Under the assumption that you were connected to honest nodes before the attack, this aims to reduce an eclipse attack from being successful.
+When your node connects to the Bitcoin network, it makes at least [two outbound block-relay only connections](https://github.com/bitcoin/bitcoin/pull/15759). This release introduces [Anchor Connections](https://github.com/bitcoin/bitcoin/pull/17428). Anchors are the [two outbound block-relay connections]() your node is connected to, logged to an `anchors.dat` file so that they can be used upon a node restart. Under the assumption that you were connected to honest nodes before the attack, this aims to reduce an eclipse attack from being successful.
```
Comment (adamjonas) [inline docs/v0.21/testing-0.21rc2.md:122]: I prefer the term block-relay-only connections. Full conns are also block-relay.

This second link is empty and I think can safely be removed.

File: docs/v0.21/testing-0.21rc2.md (hunk 1 above)
Comment (jarolrod) [reply docs/v0.21/testing-0.21rc2.md:44]: addressed in 272bb8d46bfc4ad2b05e12f735f1a4d0fbfdd903, gave co-author credit

File: docs/v0.21/testing-0.21rc2.md (hunk 2 above)
Comment (jarolrod) [reply docs/v0.21/testing-0.21rc2.md:110]: removed links in 272bb8d46bfc4ad2b05e12f735f1a4d0fbfdd903

**Pull request closed without merging**